flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/databases/clickhouse/dialect/ClickHouseDialectTypeTest.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.flink.connector.jdbc.databases.clickhouse.dialect;
+
+import org.apache.flink.connector.jdbc.dialect.JdbcDialectTypeTest;
+
+import java.util.Arrays;
+import java.util.List;
+
+/** The Clickhouse params for {@link JdbcDialectTypeTest}. */
+public class ClickHouseDialectTypeTest extends JdbcDialectTypeTest {
+
+    @Override
+    protected String testDialect() {
+        return "clickhouse";
+    }
+
+    @Override
+    protected List<TestItem> testData() {
+        return Arrays.asList(
+                createTestItem("CHAR"),
```

Comment:
Based on https://clickhouse.com/docs/en/sql-reference/data-types
ClickHouse also supports `MAP`, `ARRAY`, nested
can we support them as well?

Reply:
not support

Reply:
> Based on https://clickhouse.com/docs/en/sql-reference/data-types ClickHouse also supports `MAP`, `ARRAY`, nested can we support them as well?
not support

Reply:
it would be great if share more info about reasons

Reply:
Do you have a better idea for this question @snuyanzin